### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by rowboat85** [flurinduerst/WPDistillery#51]
Hi I am up to the command vagrant up and its displaying this error - default: [vagrant-hostsupdater] This operation requires administrative access. You may skip it by manually adding equivalent entries to the hosts file.

?

**Comment by flurinduerst** [comment]
Hi @rowboat85 

What version of Vagrant, Virtualbox and vagrant-hostsupdater are you using? You're logged in as admin right? (What OS are you on btw?)

**Comment by rowboat85** [comment]
Thanks for the prompt reply. 

Here are versions of os/software:
MAC OS Sierra 10.2
Vagrant 1.8.1
VirtualBox 5.0.20
I think I have latest hostsupdater as I ran vagrant plugin install vagrant-hostsupdater

I have another WPDistillery site running that I setup a few months back on 192.168.33.10 - would I need to make IP address for new site e.g 192.168.33.11 ?
If I were to update Vagrant & VirtualBox (if this is issue) would this effect other WPDistillery site?

mm not sure about admin rights? should I be sudo chown any particular folder/package?

Also not sure if permissions have any relevance? - I have checked my /etc/hosts the user is assigned to myself and not root - should this be root?
